lib/pathresolver.js

```diff
@@ -86,7 +86,14 @@ PathResolver.prototype = {
     var parsedFrom = url.parse(mainDocUrl);
     var parsedTo = url.parse(absUrl);
     if (parsedFrom.protocol === parsedTo.protocol && parsedFrom.host === parsedTo.host) {
-      var pathname = pathPosix.relative(pathPosix.dirname(parsedFrom.pathname), parsedTo.pathname);
+      var parsedFromDir = pathPosix.dirname(parsedFrom.pathname);
+      // top-level file paths like `index.html` will return the dirname `.`.
```

Comment:
I find this comment a little hard to grok. What's a top-level path, and where do we get it?